phone/src/os/phone/hooks/state.ts

```diff
@@ -9,4 +9,8 @@ export const phoneState = {
     key: 'phoneConfig',
     default: undefined,
   }),
+  forceUpdateState: atom({
```

Comment:
Alright, lets not. As @jfrader pointed out, its better to expand the LocalStorage hook itself to initiate a recoil update rather than doing this hacky method here.